modules/administration/pages/monitoring.adoc

```diff
@@ -313,3 +319,26 @@ You can also proxy requests through the network boundary.
 Tools like PushProx deploy a proxy and a client on both sides of the network barrier and allow Prometheus to work across network topologies such as NAT.
 
 For more information on PushProx, see https://github.com/RobustPerception/PushProx.
+
+
+
+=== Reverse Proxy Set Up
```

Comment:
```suggestion
=== Reverse Proxy Setup
```

Reply:
Heh, good spot. In my defence, I originally had "Set Up Reverse Proxy" :wink: